src/popup/components/DetailsItem.vue

```diff
@@ -1,47 +1,89 @@
 <template>
   <div
     class="details-item"
+    :class="{
+      expandable,
+      expanded,
+    }"
   >
-    <div class="label">
+    <Component
+      :is="expandable ? 'BtnPlain' : 'div'"
+      class="label"
+      @click="toggleExpanded()"
+    >
       {{ label }}
       <span
         v-if="$slots.label"
         :class="{ indent: label }"
       >
         <slot name="label" />
       </span>
-    </div>
-    <div
-      class="value"
-      :class="{ small, highlight }"
-    >
-      {{ value }}
-      <span
-        v-if="secondary"
-        class="secondary"
+
+      <ArrowIcon
+        v-if="expandable"
+        class="expand-arrow"
+      />
+    </Component>
+
+    <Transition name="fade-transition">
+      <div
+        v-if="expanded"
+        class="value"
+        :class="{ small, highlight }"
       >
-        {{ secondary }}
-      </span>
-      <slot name="value" />
-    </div>
+        <slot>
+          {{ value }}
+          <span
+            v-if="secondary"
+            class="secondary"
+          >
+            {{ secondary }}
+          </span>
+          <slot name="value" />
+        </slot>
```

Comment:
Should there be a slot inside a slot?

Reply:
In general: yes, it is normal to have slot inside of an slot. Maybe not the best solution, but commonly used in some frameworks.
In our case my goal is to migrate the way how we pass the content to `DetailsItem`. Currently we need to use named slot, but in my opinion this should be the default one. So instead having to:
```
<DetailsItem label="Test">
  <template #value>
    Some content
  </template>
</DetailsItem>
```
we should be able to:
```
<DetailsItem label="Test">
    Some content
</DetailsItem>
```

I didn't want to refactor whole app, so for now I'm leaving the named slot, but we will gradually migrate the existing code.